### ISSUE TITLE
Some undefined indexes PHP Notices in debug log

### ISSUE THREAD

**Issue by gagan0123** [rtCamp/nginx-helper#144]
On a new subdomain based multisite installation having Nginx Helper plugin activated globally on network I'm getting some PHP Notices regarding use of undefined indexes. Here's some more info:

Nginx Helper Version : 1.9.10
WordPress Version 4.8.2

Error Log Messages:
```
[21-Oct-2017 07:35:17 UTC] PHP Notice:  Undefined index: log_level in /home/***/***/htdocs/wp-content/plugins/nginx-helper/admin/lib/nginx-general.php on line 428
[21-Oct-2017 07:35:17 UTC] PHP Notice:  Undefined index: log_level in /home/***/***/htdocs/wp-content/plugins/nginx-helper/admin/lib/nginx-general.php on line 429
[21-Oct-2017 07:35:17 UTC] PHP Notice:  Undefined index: log_level in /home/***/***/htdocs/wp-content/plugins/nginx-helper/admin/lib/nginx-general.php on line 430
[21-Oct-2017 07:35:17 UTC] PHP Notice:  Undefined index: log_level in /home/***/***/htdocs/wp-content/plugins/nginx-helper/admin/lib/nginx-general.php on line 431
[21-Oct-2017 07:35:17 UTC] PHP Notice:  Undefined index: log_filesize in /home/***/***/htdocs/wp-content/plugins/nginx-helper/admin/lib/nginx-general.php on line 438
```

**Comment by mattbartel** [comment]
Same issue here on the same versions of Nginx Helper and WP.

**Comment by chandrapatel** [comment]
This has been fixed in version 2.0.0 so closing it.
